reflex-dom/reflex-dom.cabal

```diff
@@ -131,7 +131,8 @@ library
     , Reflex.Dom.Widget.Resize
     , Reflex.Dom.Xhr
     , Reflex.Dom.Xhr.FormData
-  ghc-options: -Wall -fwarn-tabs -funbox-strict-fields -O2 -ferror-spans
+  ghc-options: -Wall -fwarn-tabs -funbox-strict-fields -O2 -ferror-spans -fPIC
+  cc-options: -fPIC
```

Comment:
We should have a TODO here. reflex platform should take care of this and it shouldn't go on Hackage.